source/Halibut.Tests/Transport/ConnectionManagerFixture.cs

```diff
@@ -20,7 +21,11 @@ public class ConnectionManagerFixture
         {
             MessageExchangeProtocol ExchangeProtocolBuilder(Stream s, ILog l) => GetProtocol(s, l, syncOrAsync);
 
-            var connection = new SecureConnection(Substitute.For<IDisposable>(), Stream.Null, ExchangeProtocolBuilder, Substitute.For<ILog>());
+            AsyncHalibutFeature asyncHalibutFeature = syncOrAsync == SyncOrAsync.Sync ? AsyncHalibutFeature.Disabled : AsyncHalibutFeature.Enabled;
```

Comment:
This will conflict with my PR. I've already dealt with one conflict. Can we please merge this after my PR goes in? 🙏